Daytime-Operations/Auxiliary-Telescope/AT-Calibrations/prepare-atcs-for-flat-fields-procedure.rst

```diff
@@ -8,7 +8,54 @@
 Prepare ATCS For Flat Fields Procedure
 ######################################
 
-Under construction.
+Overview
+========
+
+This procedure describes how to execute flat field calibration with LATISS during day time checkout procedure, using different filters.
```

Comment:
This page describes how to prepare ATCS (Auxiliary Telescope Control System) that includes the telescope and dome for the calibrations, but the calibrations are described in a different page.

Reply:
I think was is being asked here is to provide a link to another page that describes the daytime calibrations?